strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/AuthorizationConfigControllerTest.java

```diff
@@ -1,64 +1,405 @@
 package org.carlspring.strongbox.controllers;
 
 import org.carlspring.strongbox.config.IntegrationTest;
+import org.carlspring.strongbox.forms.PrivilegeForm;
+import org.carlspring.strongbox.forms.PrivilegeListForm;
+import org.carlspring.strongbox.forms.RoleForm;
+import org.carlspring.strongbox.forms.RoleListForm;
 import org.carlspring.strongbox.rest.common.RestAssuredBaseTest;
 import org.carlspring.strongbox.security.Role;
 import org.carlspring.strongbox.users.domain.Privileges;
+import org.carlspring.strongbox.users.security.AuthorizationConfig;
 import org.carlspring.strongbox.users.security.AuthorizationConfigProvider;
 
 import javax.inject.Inject;
-import java.util.Arrays;
-import java.util.HashSet;
+import java.util.*;
 
-import io.restassured.module.mockmvc.RestAssuredMockMvc;
+import com.google.common.collect.Sets;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import org.springframework.test.context.junit4.SpringRunner;
+import static io.restassured.module.mockmvc.RestAssuredMockMvc.given;
+import static org.carlspring.strongbox.controllers.AuthorizationConfigController.*;
+import static org.hamcrest.CoreMatchers.containsString;
 
+/**
+ * @author Pablo Tirado
+ */
 @IntegrationTest
-@RunWith(SpringJUnit4ClassRunner.class)
+@RunWith(SpringRunner.class)
```

Comment:
@ptirador ,

I'm sorry if I've asked you this already and forgotten, (I've seen this across your other pull requests), but what is the difference between these two annotations? (This is just for my better understanding).

Reply:
SpringRunner is an alias for the SpringJUnit4ClassRunner.
It's just a bit easier on the eye, don't you think?

Reply:
Right, I see! Thanks for the clarification!